docs/DeveloperGuide.md

```diff
@@ -1007,13 +1007,13 @@ The overview of the design on schedule features can be found [here](#schedule-re
 #### Update Schedule
 A summary of the general procedure of updating a plan for a particular day to the schedule in WerkIt! is as follows:
 1. User enters the command `schedule /update <day number> <plan number>`.
-2. If there are no plan being scheduled for the day, a new Day object is created and stored in the application.
-   If there is an existing plan scheduled for that particular day, the Day object that had already been created,
+2. If there is no plan being scheduled for the day, a new Day object is created and stored in the application.
+   If there is an existing plan scheduled for that particular day, the `Day` object that has already been created,
    will then be updated to store the latest plan scheduled for the day.
-3. The success response is printed to the user through the terminal.
-4. The Day object data is written to the resource file `schedule.txt`.
+3. The success response is displayed via the terminal.
+4. The `Day` object data is written to the resource file `schedule.txt`.
 
-The following sequence illustrates how the schedule /update command works in greater detail:
+The following sequence illustrates how the schedule /update command works in greater details:
```

Comment:
I think the previous version "...in greater detail" sounds better?

Also, is the phrase "schedule /update" meant to be marked up?